File: AndroidLint/library/src/main/java/com/github/mag0716/library/view/CustomView.java
package com.github.mag0716.library.view;

import android.content.Context;
import android.support.annotation.Nullable;
import android.util.AttributeSet;
import android.view.View;

/**
 * Custom views in libraries should use res-auto-namespace
 * <p>
 * Created by mag0716 on 2017/10/28.
 */
public class CustomView extends View {
    public CustomView(Context context) {
        super(context);
    }

    public CustomView(Context context, @Nullable AttributeSet attrs) {
        super(context, attrs);
    }

    public CustomView(Context context, @Nullable AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
    }
}

File: Sample/LocaleListSample/app/src/main/java/com/github/mag0716/localelistsample/CustomView.kt
package com.github.mag0716.localelistsample

import android.content.Context
import android.util.AttributeSet

class CustomView @JvmOverloads constructor(
    context: Context,
    attrs: AttributeSet? = null,
    defStyleAttr: Int = 0
) : androidx.appcompat.widget.AppCompatTextView(context, attrs, defStyleAttr) {

    init {
        setText(R.string.custom_view_text)
    }
}